cmd/cmd.go

```diff
@@ -59,6 +60,24 @@ func GetMeshConfig(kube kubernetes.Interface, namespace, name string) (*proxycon
 		return nil, multierror.Prefix(err, "failed to convert to proto.")
 	}
 
+	if mesh.MixerAddress != "" && mesh.StatsdUdpAddress == "" {
+		colon := strings.Index(mesh.MixerAddress, ":")
+		mixerAddr := mesh.MixerAddress[:colon]
+		if err := model.ValidateIPv4Address(mixerAddr); err == nil {
+			mesh.StatsdUdpAddress = mesh.MixerAddress
+		} else {
+			if mixerSvc, err := kube.CoreV1().Services(namespace).Get(mixerAddr, v1.GetOptions{}); err == nil {
```

Comment:
Without this, it makes configuring the export much harder. I'm open to good ideas, however.

Envoy wants an IP, not a service name. So, to get this to work, something needs to map from say mixer_server to ip. It seems easier to have cmd do this than it is to have an operator deploy mixer first, get the IP, then update the mesh file, then deploy pilot.

But, yes, if there is a better way, let's shift to that.